src/commands/prepare.ts

```diff
@@ -160,6 +160,21 @@ export const prepareDeployables = async (
         writeUpdatedDeployable(deployable, disableDocs),
       ),
     );
+    const staged = shell.exec('git diff --name-only --cached')
+      .toString().split('\n').filter(Boolean);
+    for (const deployable of dirtyDeployables) {
+      try{
+        const rootPath: string = shell.exec('git rev-parse --show-toplevel', {silent:true}).toString('utf8');
+        const deployableName = deployable.file.slice(rootPath.length);
```

Comment:
This may not be a safe operation if the deployable.file path does not contain the top level path. Probably better to do something like this:
```suggestion
        const deployableName = deployable.file.replace(rootPath, '');
```